library/CMakeLists.txt

```diff
@@ -255,11 +255,11 @@ set(CPACK_PACKAGE_NAME "rocrand")
 set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc")
 set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Saad Rahim <saad.rahim@amd.com>")
 if(HIP_PLATFORM STREQUAL "hcc")
-  set(CPACK_DEBIAN_PACKAGE_DEPENDS "hip_hcc (>= 1.5.19055)") # 1.5.19055 is part of ROCm 2.2
-  set(CPACK_RPM_PACKAGE_REQUIRES "hip_hcc >= 1.5.19055") # 1.5.19055 is part of ROCm 2.2
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-dev (>= 1.5.19055)") # 1.5.19055 is part of ROCm 2.2
+  set(CPACK_RPM_PACKAGE_REQUIRES "rocm-dev >= 1.5.19055") # 1.5.19055 is part of ROCm 2.2
```

Comment:
version number is wrong. Please use  2.5.27. Let's just stick with what Nico used.